### ISSUE TITLE
Trigger automatic actions defined by Hydra events in delegate server

### ISSUE THREAD

**Issue by uhbif19** [mlabs-haskell/hydra-auction#174]
This task requires to introduce new thread placing in queue HydraEvents which are triggered by:

* `DelegateEvent HydraEvent`

For that, probably, some new utilities for Hydra responses monitoring are required.

Actual implementation of responses to them is out of scope of this task.

Blocked by #125
Blocked by https://github.com/mlabs-haskell/hydra-auction/issues/140
